### ISSUE TITLE
Establish permissions and access

### ISSUE THREAD

**Issue by mcarrano** [rh-uxd/uxd-hub#14]
Identify access privileges for different areas of UXD Hub. Consider both who can view content, i.e. what is public and what is restricted to users who are logged in, and who has permissions to add or edit content living in different sections of the site.

**Comment by matthewcarleton** [comment]
I think this is done now right Matt?

**Comment by mcarrano** [comment]
Yes, although it was pointed out to me yesterday that people outside of Red Hat can still see linked resources on Google Drive and request access from the owner. This may or may not be considered a problem, but it came as a surprise to the person who brought it up to me. Perhaps we should discuss at the next Admin call.
